### ISSUE TITLE
command repeat fails when pressing enter in "target remote" mode

### ISSUE THREAD

**Issue by ckxckx** [pwndbg/pwndbg#577]
### Description

Using ENTER to repeat previous command is OK when I debug with pwndbg locally. However, when I try to debug kernel by "target remote: port", the repeat function goes fail.

You see, when pwndbg is disabled, raw gdb's command repeat function still works well in "target remote" mode. 

So, is there any thing to do with the bug?

ps: Although I test gdb on Mac OS now, I believe the bug can be reproduce on ubuntu that I had tried it half year ago. 

### My setup

When you have a kernel image:
1. gdb
2. file ./vmlinux-4.4.0-139-generic
3. target remote:8864

Then you want to debug step-by-step, but "si" can't repeat by ENTER 

If it is somehow unavailable, use:
* `show version` - for gdb
```
pwndbg> show version
GNU gdb (GDB) 8.2
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-apple-darwin17.4.0".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word".
```

* `py import sys; print(sys.version)` - for python

```
pwndbg> py import sys; print(sys.version)
2.7.10 (default, Jul 15 2017, 17:16:57)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)]
```
* pwndbg version/git commit id
```
commit df32e12583c6b64099bc958b3c692288815f75b9 (HEAD -> dev, origin/dev, origin/HEAD)
Author: Gyorgy Miru <mirugy@gmail.com>
Date:   Fri Nov 23 02:34:26 2018 +0100

    Fixed value exception on gdb 8.2 (#575)

    * Fixed value exception on gdb 8.2

    * More explicit comment

commit 2978cd86bc67a13ade7b98241d7160b076aa8f3c
Author: Gyorgy Miru <mirugy@gmail.com>
Date:   Thu Nov 22 23:04:52 2018 +0100
```

**Comment by FrancescoMagliocco** [comment]
I can confirm that this is an issue when just debugging regularly as well.  It's rather annoying, but seems like it could be a simple fix.  Really looking forward to a fix for this.

**Comment by disconnect3d** [comment]
@FrancescoMagliocco Can you give some more details? Repeating commands with enter works for me.

I believe this bug is related to #523 - i.e. it happens only on gdb 8.2.

**Comment by disconnect3d** [comment]
@ckxckx It works for me with gdb 8.1. I believe this is related to gdb 8.2 bug.

**Comment by FrancescoMagliocco** [comment]
I'm trying to downgrade gdb but I can't seem to find the exact string version to use with `apt-get`.  I've done `apt-cache showpkg gdb` but it just shows gdb 8.2-1

Quickly just removing the .gdbinit from ~ on Debian/Buster.  Commands seem to be repeating.

When I updated pwndbg, all I did was pull from the repo, and ran setup.sh again..  Could that be the reason why commands are not being repeated?

**Comment by disconnect3d** [comment]
@FrancescoMagliocco I usually build gdb by myself. The steps are more or less (adjust e.g. python path or number of CPUs to use for compilation (the `-j` flag to `make`)):
```
wget https://ftp.gnu.org/gnu/gdb/gdb-8.1.tar.gz
tar -xvf gdb-8.1.tar.gz
cd gdb-8.1
mkdir build
cd build
../configure --prefix=/usr --disable-nls --disable-werror --with-system-readline --with-python=/usr/bin/python3.6 --with-system-gdbinit=/etc/gdb/gdbinit --enable-targets=all
make -j7
sudo make install
```

**Comment by disconnect3d** [comment]
Shall be fixed on gdb 8.2.2
